src/Microsoft.FeatureManagement/ServiceCollectionExtensions.cs

```diff
@@ -58,6 +58,12 @@ public static IFeatureManagementBuilder AddFeatureManagement(this IServiceCollec
 
             services.TryAddScoped<IVariantFeatureManagerSnapshot>(sp => sp.GetRequiredService<FeatureManagerSnapshot>());
 
+            services.AddSingleton<PercentageFilter>();
```

Comment:
We probably shouldn't add PercentageFilter. It was created to showcase how to write a feature filter. We may want to drop it one day.

Reply:
Discussed with @zhenlan offline. My concern is that if we have the filter in the library, it makes sense to add it by default. If we want to exclude it by default, then most likely we should just remove it from the library as well.

Given that we do have current usage of this filter, it doesn't make sense to remove it at this point. We agreed that we should add the percentage filter by default.